server/schemas/field.graphql

```diff
@@ -97,6 +98,8 @@ type SchemaFieldInteger {
 
 type SchemaFieldReference {
   modelId: ID!
+  correspondingField: SchemaField
+  correspondingFieldId: ID
```

Comment:
```suggestion
  correspondingFieldId: ID!
```

required?

Reply:
I think that this field should be optional because it is only required in the two-way reference fields!